content/faq/serverless.md

```diff
@@ -191,13 +191,27 @@ service: serverless-example
 plugins:
   - serverless-offline
 
+package:
+  patterns:
+    # here you can ignore some files or folders to reduce the bundle size.
+    - '!node_modules/**/LICENSE'
+    - '!node_modules/**/*.md'
+    - '!node_modules/**/*.js.map'
+    - '!node_modules/**/*.d.ts.map'
+    - '!node_modules/**/*.d.ts'
+    - '!node_modules/typeorm/browser/**'
+    - '!.webpack/**'
+    - '!src/**'
+    - '!test/**'
+    - '!.idea/**'
```

Comment:
This change doesn't seem to be relevant to NestJS integration. Note: this chapter isn't supposed to serve as a "complete guide to Serverless with Node" but rather showcase how you can use NestJS with serverless, without diving into non-framework-relevant details.

Reply:
But when we explain how to integrate with AWS Lambda using libraries like `serverless framework` and `@vendia/serverless-express` we already do that. But now, the example is more complete, fixing issues and providing some optimizations.

But if you still think it's not appropriate, I can remove all the node_modules specs and just leave `src`, `test` and `.webpack` because those folders are added by NestJS and the integration, or I can move to the runtime optimization, which do you prefer?

Reply:
Also, this optimization itself can reduce the package size from 9.9mb to 4.9mb. I know this can be specific but isn't the amount of bundle that could be saved worth leaving as is?

For my own experiences, this optimization is much better than running webpack to bundle the entire nest itself.